### DISCUSSION TITLE
在翻译时，应当保留原文的断句位置还是进行一些修改

### DISCUSSION THREAD

**Discussion by baka-gourd** [CFPAOrg/Minecraft-Mod-Language-Package#1172] [Q&A] [answered]
来源：https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/pull/1170
这个需要讨论一下。

**Comment by Determancer** [comment]
刚才谈及的不是语序问题，而是语句节奏问题，或者说是断句问题。
建议改为“应当保留原文的断句位置还是进行一些修改”。

**Comment by baka-gourd** [comment]
已更改。我认为应该考虑到中英习惯的不同，更偏向与中文的断句，毕竟是给中国人看的。

**Reply by Determancer** [reply]
针对具体的那个句子，这里有一个类似的句子：
”他的情绪很不稳定，因为他长年睡眠不足，而小明为此吃了苦头。“
同样是”<陈述>，因为……，而……“
不知道在这个句子里，这种断句是否显得奇怪？

**Comment by WuzgXY-GitHub** [comment, marked as answer]
为什么还要讨论这个，答案显而易见是适当修改啊

**Reply by WuzgXY-GitHub** [reply]
https://zh.wikipedia.org/wiki/Wikipedia:翻译腔
这个真的可以解决很多问题

**Comment by baka-gourd** [comment]
https://zh.wikipedia.org/wiki/%E6%AD%90%E5%8C%96%E4%B8%AD%E6%96%87
